docs/synapse/userguides/ug016_storm_ref_byhandlers.rst

```diff
@@ -0,0 +1,12 @@
+Storm Reference - "By" Handlers
+===============================
+Todo
+
+
+See the `Storm Syntax Conventions`__ for an explanation of the usage format used below.
+
+Where specific query examples are given, they are commonly provided in pairs using operator syntax followed by the equivalent macro syntax (if available).
+
+
+.. _conventions: ../userguides/ug011_storm_basics.html#syntax-conventions
+__ conventions_
```

Comment:
This page looks good to me as long as the PRs will be done in another PR (and weren't intended to be done in this one)